<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui3ds2pie</class>
 <widget class="QDialog" name="Gui3ds2pie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3ds2pie</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="inputFile_edit">
       <property name="toolTip">
        <string>Path to the 3DS file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="outputFile_edit">
       <property name="toolTip">
        <string>Path to the PIE file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="texturePage_edit">
       <property name="toolTip">
        <string>Texture page to use</string>
       </property>
       <property name="prefix">
        <string>page-</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="scale_slider">
       <property name="toolTip">
        <string>Scale for model</string>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="inputFile_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="outputFile_browse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="scale_spinbox">
       <property name="toolTip">
        <string>Scale for model</string>
       </property>
       <property name="maximum">
        <double>99.989999999999995</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="inputFile">
       <property name="text">
        <string>Input file</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputFile">
       <property name="text">
        <string>Output file</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="texturePage">
       <property name="text">
        <string>Texture page</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="scale">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="swapYZ">
       <property name="toolTip">
        <string>Assume that 'Z' denotes &quot;up&quot; direction
(Should be enabled for 3DS Max created files)</string>
       </property>
       <property name="text">
        <string>Swap Y/Z</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="reverseWinding">
       <property name="toolTip">
        <string>Reverse the winding of all polygon faces
(Should be enabled for 3DS Max created files)</string>
       </property>
       <property name="text">
        <string>Reverse polygon winding</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="invertUV">
       <property name="toolTip">
        <string>Invert the vertical point of origin for UV coordinates
(Should be enabled for 3DS Max created files)</string>
       </property>
       <property name="text">
        <string>Invert U/V</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="twoSidedPolys">
       <property name="toolTip">
        <string>Use both sides of Polygons
(Not used in Warzone 2100)</string>
       </property>
       <property name="text">
        <string>Two sided polygons</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="exportPIE3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Enable floating points in PIE format.</string>
       </property>
       <property name="text">
        <string>Export PIE3</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="useTCMask">
       <property name="toolTip">
        <string>Use TCMask feature instead of old team coloring frames.</string>
       </property>
       <property name="text">
        <string>Enable TCMask texture</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Gui3ds2pie</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Gui3ds2pie</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
